src/python/pants/backend/python/lint/docformatter/rules.py

```diff
@@ -85,25 +86,25 @@ async def setup(
     )
 
     all_source_files_request = Get[SourceFiles](
-        AllSourceFilesRequest(field_set.sources for field_set in request.field_sets)
+        AllSourceFilesRequest(field_set.sources for field_set in request.request.field_sets)
     )
     specified_source_files_request = Get[SourceFiles](
         SpecifiedSourceFilesRequest(
-            (field_set.sources, field_set.origin) for field_set in request.field_sets
+            (field_set.sources, field_set.origin) for field_set in request.request.field_sets
         )
     )
 
     requests: List[Get] = [requirements_pex_request, specified_source_files_request]
-    if request.field_sets.prior_formatter_result is None:
+    if request.request.prior_formatter_result is None:
         requests.append(all_source_files_request)
     requirements_pex, specified_source_files, *rest = cast(
         Union[Tuple[Pex, SourceFiles], Tuple[Pex, SourceFiles, SourceFiles]],
         await MultiGet(requests),
     )
 
     all_source_files_snapshot = (
-        request.field_sets.prior_formatter_result
-        if request.field_sets.prior_formatter_result
+        request.request.prior_formatter_result
```

Comment:
Oh, hm. This is weird naming. It'd be great to rename the parameter to `setup_request` and rename the property `SetupRequest.request` to `SetupRequest.docformatter_request`. Ditto on Black and Isort.